docs/get-started/features.md

```diff
@@ -137,519 +105,111 @@ See the full list in [github.com/klauspost/cpuid][klauspost-cpuid].
-| version | full                | Full kernel version as reported by `/proc/sys/kernel/osrelease` (e.g. '4.5.6-7-g123abcde')
-|         | major               | First component of the kernel version (e.g. '4')
-|         | minor               | Second component of the kernel version (e.g. '5')
-|         | revision            | Third component of the kernel version (e.g. '6')
+| Feature | Value  | Description
+| ------- | ------ | -----------
+| **`kernel-config.<option>`**  | true   | Kernel config option is enabled (set 'y' or 'm'). Default options are `NO_HZ`, `NO_HZ_IDLE`, `NO_HZ_FULL` and `PREEMPT`
+| **`kernel-selinux.enabled`**  | true   | Selinux is enabled on the node
+| **`kernel-version.full`**     | string | Full kernel version as reported by `/proc/sys/kernel/osrelease` (e.g. '4.5.6-7-g123abcde')
+| **`kernel-version.major`**    | string | First component of the kernel version (e.g. '4')
+| **`kernel-version.minor`**    | string | Second component of the kernel version (e.g. '5')
+| **`kernel-version.revision`** | string | Third component of the kernel version (e.g. '6')
 
-Kernel config file to use, and, the set of config options to be detected are
-configurable. See [configuration](deployment-and-usage#worker-configuration)
-for more information.
+The kernel label source is configurable, see
+[worker configuration](deployment-and-usage#worker-configuration) and
+[`sources.kernel`](../advanced/worker-configuration-reference#sourceskernel)
+configuration options for details.
 
 ### Memory
 
-The **memory** feature source supports the following labels:
-
-| Feature | Attribute | Description                                            |
-| ------- | --------- | ------------------------------------------------------ |
-| numa    |           | Multiple memory nodes i.e. NUMA architecture detected
-| nv      | present   | NVDIMM device(s) are present
-| nv      | dax       | NVDIMM region(s) configured in DAX mode are present
+| Feature     | Value | Description
+| ----------- | ----- | -----------
+| **`memory-numa`**       | true | Multiple memory nodes i.e. NUMA architecture detected
+| **`memory-nv.present`** | true | NVDIMM device(s) are present
+| **`memory-nv.dax`**     | true | NVDIMM region(s) configured in DAX mode are present
 
 ### Network
 
-The **network** feature source supports the following labels:
-
-| Feature | Attribute  | Description                                           |
-| ------- | ---------- | ----------------------------------------------------- |
-| sriov   | capable    | [Single Root Input/Output Virtualization][sriov] (SR-IOV) enabled Network Interface Card(s) present
-|         | configured | SR-IOV virtual functions have been configured
+| Feature     | Value | Description
+| ----------- | ----- | -----------
+| **`network-sriov.capable`**    | true | [Single Root Input/Output Virtualization][sriov] (SR-IOV) enabled Network Interface Card(s) present
```

Comment:
did you add some new labels for `network`?

Reply:
Nope, this has been there for a while, already

Reply:
I was referring to #660 but after taking a closer look it did not change the existing labels.